src/features/DisplayControls/DrugChart/utils/DrugChartUtils.js

```diff
@@ -106,15 +107,19 @@ export const transformDrugOrders = (orders) => {
   return medicationData;
 };
 
-const isLateTask = (startTime, drugChart) => {
+export const isLateTask = (startTime, drugChart) => {
   const currentTime = Math.floor(new Date().getTime() / 1000);
   const lateTaskStatusWindowInSeconds =
     drugChart.timeInMinutesFromNowToShowPastTaskAsLate * 60;
 
   return startTime < currentTime - lateTaskStatusWindowInSeconds;
 };
 
-const isAdministeredLateTask = (startTime, effectiveStartDate, drugChart) => {
+export const isAdministeredLateTask = (
```

Comment:
That should be fine, because the function which is calling the function is already having the config fetched. So moving it to Common Utils should be fine as this could be re-used elsewhere.